spec/custom/framework.js

```diff
@@ -4,6 +4,5 @@
  *
  * @param {Runner} runner The current Protractor Runner.
  * @param {Array} specs Array of Directory Path Strings.
- * @return {q.Promise} Promise resolved with the test results
  */
-exports.run = require('../../lib/frameworks/jasmine.js').run;
+exports.run = require('../../built/frameworks/jasmine').run;
```

Comment:
```suggestion
exports.run = require('../../built/lib/frameworks/jasmine').run;
```

The lib path still is required.

Reply:
Hm, when I made `tsc` there is no `lib` folder inside the `built`. Also, I checked that with current path it works.

Reply:
Yup. I agree.